Sources/SnapshotTesting/Common/View.swift

```diff
@@ -754,12 +757,16 @@ private func add(traits: UITraitCollection, viewController: UIViewController, to
       viewController.view.trailingAnchor.constraint(equalTo: rootViewController.view.trailingAnchor),
       ])
   }
-  rootViewController.addChild(viewController)
   rootViewController.setOverrideTraitCollection(traits, forChild: viewController)
   viewController.didMove(toParent: rootViewController)
+  window.rootViewController = rootViewController
+
   rootViewController.beginAppearanceTransition(true, animated: false)
   rootViewController.endAppearanceTransition()
-  window.rootViewController = rootViewController
+
+  viewController.view.setNeedsLayout()
+  viewController.view.layoutIfNeeded()
+
   rootViewController.view.setNeedsLayout()
   rootViewController.view.layoutIfNeeded()
```

Comment:
Is the root VC layout step still needed?

Reply:
Doesn't seem like it, but feels like a safe thing to keep in?